speech/snippets/speech_adaptation_beta.py

```diff
@@ -80,20 +80,5 @@ def sample_recognize(storage_uri: str, phrase: str) -> speech.RecognizeResponse:
     return response
 
 
-def main() -> None:
-    import argparse
-
-    parser = argparse.ArgumentParser()
-    parser.add_argument(
-        "--storage_uri",
-        type=str,
-        default="gs://cloud-samples-data/speech/brooklyn_bridge.mp3",
-    )
-    parser.add_argument("--phrase", type=str, default="Brooklyn Bridge")
-    args = parser.parse_args()
-
-    sample_recognize(args.storage_uri, args.phrase)
-
-
 if __name__ == "__main__":
-    main()
+    recognition_response = sample_recognize()
```

Comment:
Why store response in `recognition_response` ?

Reply:
I agree. It is not necessary.